tests/tests/parallel_restore.sh

```diff
@@ -12,9 +12,6 @@ echo "== make small meta fs"
 # meta device just big enough for reserves and the metadata we'll fill
 scoutfs mkfs -V 2 -A -f -Q 0,127.0.0.1,53000 -m 10G -d 60G "$T_EX_META_DEV" "$T_EX_DATA_DEV" > $T_TMP.mkfs.out 2>&1 || \
 	t_fail "mkfs failed"
-mount -t scoutfs -o metadev_path=$T_EX_META_DEV,quorum_slot_nr=0 \
-	"$T_EX_DATA_DEV" "$SCR"
-umount "$SCR"
```

Comment:
I have a separate commit to fix this bug - we should split it out because we can squash it away before it gets merged into `main` eventually.

Reply:
Hmm, github isn't making it too clear what you're referencing here, could you elaborate a bit?